modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -226,6 +235,9 @@ private ApiResponses _getApiResponses(
 			schemaName = OpenAPIContributorUtil.getPageSchemaName(
 				relatedObjectDefinition);
 		}
+		else if (StringUtil.equals(httpMethod, "delete")) {
```

Comment:
Maybe it would be better to include the else if in the next clause to get the Schema name and set it to null by default (that in this case, it would be that case)